### PR TITLE
Update executable convention, avoids conflict with binstubs

### PR DESCRIPTION
**Pull request by bf4**
ref: http://bundler.io/blog/2015/03/20/moving-bins-to-exe.html and https://github.com/bundler/bundler/commit/ab3e21784c6c18702869c771fbe7ae23c82cc7c0

### CODE REVIEW

**Comment by portertech** [comment]
Do we care? I'm not aware of a valid use of bin stubs for Sensu.

**Comment by bf4** [comment]
Up to you.  If someone `bundle binstub rake` or `bundle binstub rspec-core` etc., and commits them, they'd get included in the gem.  This PR has no effect on usage of the gem.  Feel free to close.

**Comment by portertech** [comment]
@bf4 does a gem install still place the gem executables in `exe/` in the Ruby bin path?

File: sensu.gemspec [hunk 1]
```diff
@@ -34,6 +34,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency "em-http-request", "~> 1.1"
 
   s.files         = Dir.glob("{bin,lib}/**/*") + %w[sensu.gemspec README.md CHANGELOG.md MIT-LICENSE.txt]
-  s.executables   = Dir.glob("bin/**/*").map { |file| File.basename(file) }
+  s.executables   = Dir.glob("exe/**/*").map { |file| File.basename(file) }
```
Comment (bf4) [inline sensu.gemspec]: when installing the sensu gem, the files in `exe` will be installed as executables (just like they were before when in bin), per `spec.executables`

Comment (portertech) [inline sensu.gemspec]: @bf4 cool, thank you for the clarification :+1: I have kicked the failed build, will merge this PR when it goes green! Thank you.

Comment (bf4) [inline sensu.gemspec]: :smile:

**Comment by portertech** [comment]
:boom:

**Pull request merged**